ingestion/functions/parsing/new_zealand/new_zealand_test.py

```diff
@@ -47,20 +47,23 @@
 
 
 def _c(location, gender, age_start, age_end, confirmed_date, travel=False, notes=None):
+    demographics = {}
+    if age_start is not None and age_end is not None:
+        demographics["ageRange"] = {"start": age_start, "end": age_end}
+    if gender:
+        demographics["gender"] = gender
     C = {
         "caseReference": _caseReference,
         "location": location,
-        "demographics": {
-            "ageRange": {"start": age_start, "end": age_end},
-            "gender": gender,
-        },
         "events": [
             {
                 "name": "confirmed",
                 "dateRange": {"start": confirmed_date, "end": confirmed_date},
             }
-        ]
+        ],
     }
+    if demographics:
```

Comment:
this test is redundant, as demographics is definitely not None (it might be empty though)

Reply:
that checks for demographics != {}, as we don't want to add an empty demographics

Reply:
ah you're right, it does! I thought empty but existing collection would be truthy.